docs/4.0/components/buttons.md

```diff
@@ -154,4 +154,5 @@ Note that pre-checked buttons require you to manually add the `.active` class to
 
 | Method | Description |
 | --- | --- |
-| `$().button('toggle')` |Toggles push state. Gives the button the appearance that it has been activated.  |
+| `$().button('toggle')` | Toggles push state. Gives the button the appearance that it has been activated.  |
```

Comment:
Nitpicking here and in the next line there are 2 spaces before the end `|`.

Reply:
nice catch :smile: